05-Time-Series-Raster.Rmd

```diff
@@ -213,8 +205,32 @@ xres(NDVI_HARV_stack)
 
 ```
 
+Notice that the CRS is `+proj=utm +zone=19 +ellps=WGS84 +units=m +no_defs`. The
+**CRS** is in UTM Zone 19.  If you have completed the previous tutorials in 
+this 
+[raster data in `R` series ]({{ site.baseurl }}tutorial/spatial-raster-series),
+you should notice that the UTM Zone for the remote sensing non-Landsat-derived
+data was 18, not 19.  Why is it now Zone 19?  It this an error in our data?  
+
+<figure>
+    <a href="{{ site.baseurl }}/images/dc-spatial-raster/UTM_zones_18-19.jpg">
+    <img src="{{ site.baseurl }}/images/dc-spatial-raster/UTM_zones_18-19.jpg"></a>
+    <figcaption> Landsat imagery swaths are over 170 km N-S and 180 km E-W. As 
```

Comment:
The width of a Landsat scene is extremely wide - spanning 170km North to South and 180km East to West. This means that Landsat data often cover multiple UTM zones. When the data are processed, the zone in which the majority of the data cover, is the zone which is used for the final CRS. Thus our Field Site at Harvard Forest, is located in UTM Zone 18. But the Landsat data, is in a CRS of UTM Zone 19.